src/sqs.ts

```diff
@@ -31,7 +31,7 @@ export type SQSMessageHandlerConfig<Message, Context> = {
    * about SQS partial batch responses see
    * https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html#services-sqs-batchfailurereporting
    */
-  partialBatching?: boolean;
+  usePartialBatchResponses?: boolean;
```

Comment:
It might be worth adding an explicit comment to the JSDoc here, indicating to users that:
- in addition to setting this flag, they need to also explicitly configure their event source mapping to use partial batch responses